<p>Logic classes must be annotated with Stataless annotation</p>
<h2>Noncompliant Code Example</h2>
<pre>public class AuthorLogic {
    ...
}
</pre>
<h2>Compliant Solution</h2>
<pre>@Stateless
public class AuthorLogic {
    ...
}
</pre>